### PR TITLE
B 17657 too can view updated sit params on sit resubmission

### PR DESCRIPTION
**Pull request by traskowskycaci**
## Summary

When a service item is resubmitted, we want the TOO to be able to view the changed values. This allows those changed values to be displayed in a tooltip that shows the previous and current values for every changed item.

## Verification Steps for the Author

These are to be checked by the author.

- [x] Tested in the Experimental environment (for changes to containers, app startup, or connection to data stores)
- [x] Have the Agility acceptance criteria been met for this change?

## Verification Steps for Reviewers

These are to be checked by a reviewer.

- [ ] Has the branch been pulled in and checked out?
- [ ] Have the BL acceptance criteria been met for this change?
- [ ] Was the CircleCI build successful?
- [ ] Has the code been reviewed from a standards and best practices point of view?

### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Access the office application.
2. Login as a TOO.
3. Select a move with a resubmitted service item. I used move SITEXT for my testing.
4. Navigate to the Move task order page and scroll down to the Requested service items section.
5. Verify that a tooltip appears and that when hovering over the tooltip, you see the previous and current values for any details that changed during the resubmission.

### Frontend

- [x] There are no aXe warnings for UI.
- [x] This works in [Supported Browsers and their phone views](https://transcom.github.io/mymove-docs/docs/adrs/Browser-Support/#minimum-browser-requirements) (Chrome, Firefox, Edge).
- [x] There are no new console errors in the browser devtools.
- [x] There are no new console errors in the test output.
- [x] This change meets the standards for [Section 508 compliance](https://www.ssa.gov/accessibility/andi/help/install.html).

### CODE REVIEW

File: src/components/Office/ServiceItemsTable/ResubmittedServiceItemDetails.test.jsx [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+          },
+        ],
+      },
+      isSuccess: true,
+    };
+
+    useMovePaymentRequestsQueries.mockReturnValue(multiplePaymentRequests);
+    useGHCGetMoveHistory.mockReturnValue(history);
+
+    // useGHCGetMoveHistory.mockReturnValue(history);
```
Comment (deandreJones) [inline src/components/Office/ServiceItemsTable/ResubmittedServiceItemDetails.test.jsx:74]: do we need to keep this commented out code

Comment (traskowskycaci) [reply src/components/Office/ServiceItemsTable/ResubmittedServiceItemDetails.test.jsx:74]: No we don't. I noticed I have some more cleanup to do so I will take care of that.

**Comment by danieljordan-caci** [comment]
Update with main and pass all checks and this should be good to go for me!

**Review by danieljordan-caci — APPROVED** [review]
All tests are passing, tooltip shows up for appropriate resubmissions. LGTM!

**Review by cameroncaci — APPROVED** [review]

**Review by deandreJones — APPROVED** [review]

**Pull request merged**